sat_cty/city_summary.py

```diff
@@ -88,7 +88,7 @@ def calc_ndvi(dc: Dataset, red_band_name: str, nir_band_name: str) -> Dataset:
         level=logging.INFO,
     )
 
-    wkdir = sys.argv[1] if len(sys.argv) > 1 else "/data/sat-cty"
+    wkdir = sys.argv[1] if len(sys.argv) > 1 else "sat-cty/data"
```

Comment:
So the issue here is that it'll use the code's dir as the place to dump data, which is generally not good. Maybe we could use `tempfile.mkdtemp` instead: https://docs.python.org/3/library/tempfile.html#tempfile.mkdtemp

I use this pretty frequently to avoid worrying about cleaning up after myself.

Reply:
Oh I like this. Never heard of the thing before

Reply:
e.g. `test = mkdtemp()`